subprojects/docs/src/docs/userguide/jvm/jvm_test_suite_plugin.adoc

```diff
@@ -0,0 +1,165 @@
+
+Each test suite has some configuration that is common across for all tests contained in the suite:
+
+* Testing framework
+* Sources
+* Dependencies
+
+In the future, the dimensions specified in the test suite may imply which toolchains are used to compile and run tests.
+
+The `Test` task associated with the target inherits its name from the suite.  Other properties of the `Test` task are configurable.
+
+[[sec:jvm_test_suite_examples]]
+[discrete]
+== Configuration Examples
+
+Here are several examples to illustrate the configurability of test suites.
+
+== Declaring an additional test suite
+====
+include::sample[dir="snippets/testing/test-suite-plugin/groovy",files="build.gradle[tags=configure-testing-extension]"]
+include::sample[dir="snippets/testing/test-suite-plugin/kotlin",files="build.gradle.kts[tags=configure-testing-extension]"]
+====
+
+<1> Configure all test suites for this project
+<2> Configure the built-in `test` suite.  This suite is automatically created for backwards compatibility.  By convention, the built-in `test` suite will use JUnit4.
+<3> Declare this test suite uses JUnit Jupiter.  The framework's dependencies are automatically included.  It is not necessary to explicitly name or configure the built-in `test` suite if JUnit4 is desired.
+<4> Define a new suite called `integrationTest`.
+<5> Add a dependency on the production code of the project to the `integrationTest` suite targets.  By convention, only the built-in `test` suite will automatically have a dependency on the production code of the project.
+<6> Configure all targets of this suite.  By convention, test suite targets have no relationship to other `Test` tasks.  This example shows that the slower integration test suite targets should run after all targets of the `test` suite are complete.
+<7> Configure the `check` task to depend on all `integrationTest` targets.  By convention, test suite targets are not associated with the `check` task.
+
+Invoking the `check` task on the above configured build should show output similar to:
+
+[listing]
+----
+include::{snippetsPath}/testing/test-suite-plugin/tests/checkTaskOutput.out[]
+----
+
+Note that all tasks synthesized by the registration of the `integrationTest` suite (`compileIntegrationTestJava`, `processIntegrationTestResources`, `integrationTestClasses` and `integrationTest`) do not run until after the `test` suite completes its tasks.
+
+== Configuring the built-in `test` suite
+====
+include::sample[dir="snippets/testing/test-suite-configure-default-suite/groovy",files="build.gradle[tags=configure-default-suite]"]
+include::sample[dir="snippets/testing/test-suite-configure-default-suite/kotlin",files="build.gradle.kts[tags=configure-default-suite]"]
+====
+
+// TODO test with `./gradlew :docs:docsTest --tests "*test-suite-configure-default-suite*"`
+<1> All targets of the `test` test suite will use the TestNG framework
```

Comment:
it looks like you have two <1>'s in this snippet:
https://builds.gradle.org/repository/download/Gradle_Release_Check_BuildDistributions/46323550:id/distributions/gradle-7.3-docs.zip!/gradle-7.3-20210928203627%2B0000/docs/userguide/jvm_test_suite_plugin.html#configuring_the_built_in_test_suite
```suggestion
<1> Declare the `test` test suite uses the TestNG test framework.
```